src/savedSearches/form/AddOrReplaceForm.js

```diff
@@ -70,6 +70,10 @@ class AddOrReplaceForm extends React.Component {
             this.props.user.email.trim().length === 0);
     };
 
+    emailSet = () => {
+        return (this.props.user.email !== undefined && this.props.user.email !== null && this.props.user.email.trim().length !== 0);
```

Comment:
kan forenkles til return (this.props.user.email  && this.props.user.email.trim().length !== 0)